src/javascript/campaign-url-builder/components/campaign-url.js

```diff
@@ -169,44 +208,71 @@ export default class CampaignUrl extends React.Component {
               </label>
             </div>
           </div>
-          {this.renderActionsButton()}
+          {problematicElement}
+          {this.renderActionsButton(problematicElement ? true : false)}
           <div ref="url" className="u-visuallyHidden">{url}</div>
         </div>
       </div>
     );
   }
 
+  // NOTE(nathanwest): In the renderActionsButton, below, the IconButton
```

Comment:
Not sure if this note is needed. I think it's safe to assume readers of this code are familiar React.

Reply:
True, but generally keys are only used to separate elements in arrays. This would be the first time I've ever come across their use outside of an array.